pkg/cri/server/sandbox_run.go

```diff
@@ -310,6 +311,14 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 			return nil, fmt.Errorf("failed to setup network for sandbox %q: %w", id, err)
 		}
 
+		// Update metadata here to save CNI result and pod IPS to disk.
```

Comment:
nit: IPS makes it look like an acronym, but we're really talking about the plural of IP

```suggestion
		// Update metadata here to save CNI result and pod IPs to disk.
```

Reply:
Good catch, finger stuck on shift too long

Reply:
Fixed!